### DISCUSSION TITLE
Tab content blocks removes header content blocks below

### DISCUSSION THREAD

**Discussion by sierratull** [GitbookIO/community#88] [Feature Requests]
### What's your feature/idea?

When the Tabs content block is used, any content below it that is H1 and H2 items will not populate on the right-side navigation.

### What's your desired outcome?

For pages, we would like the ability to view any remaining navigation items that are placed after Tabs content block.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by aikoii** [comment]
Hi @sierratull!

We released a fix for this issue a while ago 🙂. Could you make sure you are running the latest version of our application, please? One way to do this is a hard refresh:

https://docs.gitbook.com/resources/faq/how-to-hard-refresh-your-browser

If it didn't help, could you email us at [support@gitbook.com](mailto:support@gitbook.com), so we can help you the fastest?

Thank you!
